modules/ossm-mirror-repository.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/service_mesh_install/preparing-ossm-installation.adoc
+
+[id="ossm-mirror-repository_{context}"]
+= Mirroring the {product-title} image repository
+
+Mirror the {product-title} image repository to use during cluster installation
```

Comment:
I think we need to provide context here.  This isn't something that you have to do unless you're performing a disconnected installation.  That should actually probably be the heading for this section, "Disconnected installation", or borrow the heading that the OCP docs used "Creating a mirror registry for installation in a restricted network."

Also, is there a link somewhere to the OCP topic?